code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -113,6 +113,42 @@
 	. = ..()
 	add_filter("emissives", 1, alpha_mask_filter(render_source = EMISSIVE_RENDER_TARGET, flags = MASK_INVERSE))
 	add_filter("lighting", 3, alpha_mask_filter(render_source = O_LIGHTING_VISUAL_RENDER_TARGET, flags = MASK_INVERSE))
+
+/**
+ * Renders extremely blurred white stuff over space to give the effect of starlight lighting.
+ */
+
+/atom/movable/screen/plane_master/starlight
+	name = "starlight plane master"
+	plane = STARLIGHT_PLANE
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	render_relay_plane = LIGHTING_PLANE
+	blend_mode_override = BLEND_OVERLAY
+	color = "#bcdaf7"
+
+/atom/movable/screen/plane_master/starlight/Initialize(mapload)
+	. = ..()
+	add_filter("guassian_blur", 1, gauss_blur_filter(6))
+	// Default the colour to whatever the parallax is currently
+	transition_colour(src, SSparallax.random_parallax_color, 0, FALSE)
+	// Transition the colour to whatever the global tells us to go to
+	RegisterSignal(SSdcs, COMSIG_GLOB_PARALLAX_COLOUR_CHANGE, PROC_REF(transition_colour))
+
+/atom/movable/screen/plane_master/starlight/proc/transition_colour(datum/source, new_colour, transition_time = 5 SECONDS, force_colour = FALSE)
```

Comment:
This function should be made into a global helper function and shouldn't increase the saturation if saturation is 0 (white will become red otherwise)